core/controllers/base_test.py

```diff
@@ -279,11 +279,23 @@ def test_alphabetic_i18n_keys(self):
         for filename in filenames:
             with open(os.path.join(os.getcwd(), 'assets', 'i18n', filename),
                       mode='r') as f:
-                next_line = next(f)
-                for line in f:
-                    line = line.replace("{", "")
-                    line = line.replace("}", "")
-                    self.assertGreater(next_line, line)
+                lines = f.readlines()
+                if lines[0] == '{\n':
```

Comment:
Didn't know that I had to assert it instead of using these decision structures. Going to correct all of these now. Thanks again.